src/main/java/use_cases/add_edit_expenses_use_case/ExpenseID.java

```diff
@@ -37,12 +36,52 @@ public ExpenseID(String name, Object value, String oldCategory, boolean isRecurr
         this.oldExpense = oldExpense;
     }
 
+    /**
+     * Gets String Expense name.
+     * @return String name
+     */
     public String getName(){return name;}
+
+    /**
+     * Gets Object Expense value.
+     * @return Object value from user input
+     */
     public Object getValue(){return value;}
+
+    /**
+     * Sets Double Expense value. (Casting occurs in ExpenseUCI)
+     * @param value double passed in from ExpenseUCI to cast user input into a valid Expense attribute.
+     */
     public void setValue(double value){this.value = value;}
```

Comment:
Do you need this setter?

Reply:
She does. I've asked the same thing in her other feature. I'll resolve this to save time.